handlers_test.go

```diff
@@ -0,0 +1,273 @@
+package lifecycle_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/google/go-cmp/cmp"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpacks/lifecycle"
+	"github.com/buildpacks/lifecycle/buildpack"
+	h "github.com/buildpacks/lifecycle/testhelpers"
+	"github.com/buildpacks/lifecycle/testmock"
+)
+
+func TestHandlers(t *testing.T) {
+	spec.Run(t, "Handlers", testHandlers, spec.Report(report.Terminal{}))
+}
+
+func testHandlers(t *testing.T, when spec.G, it spec.S) {
+	when("#ReadGroup", func() {
```

Comment:
I think eventually other phases will have their own config handler, and we won't need the static method.